tools/azure-sdk-tools/packaging_tools/generate_utils.py

```diff
@@ -354,6 +355,8 @@ def gen_cadl(cadl_relative_path: str, spec_folder: str) -> Dict[str, Any]:
     output_path = Path(os.getcwd()) / project_yaml["emitters"][cadl_python]["sdk-folder"]
     if not output_path.exists():
         os.makedirs(output_path)
+        while(not output_path.exists()):
+            time.sleep(1)
```

Comment:
`os.makedirs` is not sync operation so we have to make sure the path exists before running the codegen